anesthetic/weighted_pandas.py

```diff
@@ -155,131 +178,161 @@ def _constructor_expanddim(self):
 class WeightedDataFrame(_WeightedObject, DataFrame):
     """Weighted version of pandas.DataFrame."""
 
-    def mean(self, axis=0, skipna=True):
+    def _weights(self, axis):
+        return self._get_axis(axis).get_level_values('weights').to_numpy()
```

Comment:
* Should we call this `get_weights`?
* Should `axis` have a default value? Or is it better to force input to ensure correct usage?

Reply:
Agreed on both counts. Done.